playwright_dotnet_framework.BusinessLayer/Models/GetWidgetByIdModels/Result.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace playwright.dotnet.framework.BusinessLayer.Models.GetWidgetByIdModels
+{
+    public class Result
+    {
+        public int Id { get; set; }
+        public int Number { get; set; }
+        public string Name { get; set; }
+        public long StartTime { get; set; }
+        public Dictionary<string, string> Values { get; set; }
```

Comment:
`Name` and `Values` should be declared as nullable and the unneeded `System` modules should be removed